lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -1944,6 +1944,49 @@ definitions:
       records:
         $ref: '#/definitions/gateway_dns_records'
 
+  gateway_he_config:
+    description: Header Enrichment configuration
+    type: object
+    required:
+      - disable_header_enrichment
+      - enable_encryption
+      - he_encryption_algorithm
+      - he_hash_function
+      - he_encoding_type
+    properties:
+      disable_header_enrichment:
```

Comment:
@pshelar can I rename this to enable_header_enrichment? I'm not a fan of the negative flag. We could set default to True if that was your intention?

Reply:
Just indicate somewhere in the swagger.yml that the default is for he to be enabled